cartofacade/capi_test.go

```diff
@@ -113,59 +131,111 @@ func TestCGoAPI(t *testing.T) {
 		test.That(t, err.Error(), test.ShouldResemble, "VIAM_CARTO_SENSOR_NOT_IN_SENSOR_LIST")
 
 		// test invalid addSensorReading: empty reading
+		timestamp = timestamp.Add(time.Second * 2)
```

Comment:
change these back to *1

Reply:
actually we need to keep these at *1.

Basically, if I leave these all at `* 1` then I don't see cartographer changing the map at all (at least not for 10 readings). I'm not sure why this is but figure that we would prefer to have tests which have a high signal to noise ration & these tests as written show that after 1 lidar reading we have no map & a zeroed position, after 2 lidar readings we still have a zeroed position & an initial map and after 2 lidar readings we have a new map & a non zeroed position. 

If we change this to 1 second time increments we may need to call AddSensorReading a lot more times (for reasons internal to cartographer which I don't yet understand) in order to tell that same story.